app/helpers/discovered_hosts_helper.rb

```diff
@@ -66,4 +66,15 @@ def discovery_status_icon(host)
     "<span class='glyphicon #{status_glyph}' style='color: #{status_color}'
       title='#{status_message}'/>".html_safe
   end
+
+  def self.normalize_string_for_hostname(hostname)
+    hostname.to_s.downcase.gsub(/(^[^a-z0-9]|[^a-z0-9-_]|[^a-z0-9]$)/,'')
+  end
```

Comment:
That is not correct. That was correct in RFC 952, but the restriction was relaxed in RFC 1123 Section 2.1